docs/sources/tempo/setup/deployment.md

```diff
@@ -39,12 +39,11 @@ Monolithic mode can be horizontally scaled out.
 This scalable monolithic mode is similar to the monolithic mode in that all components are run within one process.
 Horizontal scale out is achieved by instantiating more than one process, with each having `-target` set to `scalable-single-binary`.
 
-This mode offers some flexibility of scaling without the configuration complexity of the full
-microservices deployment.
+This mode offers some scaling flexibility without the configuration complexity of the full microservices deployment.
 
 Each of the `queriers` perform a DNS lookup for the `frontend_address` and connect to the addresses found within the DNS record.
 
-Find a docker-compose deployment example at:
+Find a docker-compose deployment example at the following link:
```

Comment:
```suggestion
Here is a docker-compose deployment example:
```

Reply:
If you have time, changing the sentence to include a brief description of the docker-compose example with the link would be better than my suggested revision above.

Reply:
I think including the link is better as the target and method might change